openapi_python_client/openapi_parser/reference.py

```diff
@@ -21,9 +21,10 @@ class Reference:
     def from_ref(ref: str) -> Reference:
         """ Get a Reference from the openapi #/schemas/blahblah string """
         ref_value = ref.split("/")[-1]
-        class_name = stringcase.pascalcase(ref_value)
+        # ugly hack to avoid stringcase ugly pascalcase output when ref_value isn't snake case
+        class_name = stringcase.pascalcase(ref_value.replace(" ", ""))
 
         if class_name in class_overrides:
             return class_overrides[class_name]
 
-        return Reference(class_name=class_name, module_name=stringcase.snakecase(ref_value),)
+        return Reference(class_name=class_name, module_name=stringcase.snakecase(class_name),)
```

Comment:
This will conflict with #29 most likely, so whichever one gets merged first will trigger an update in the other.  Just to be aware.